eox_core/api/v1/tests/test_cohort.py

```diff
@@ -0,0 +1,101 @@
+"""
+Test module for the User Cohort API.
+"""
+from django.contrib.auth.models import User
+from django.test import TestCase
+from django.urls import reverse
+from mock import MagicMock, patch
+from rest_framework import status
+from rest_framework.test import APIClient
+
+
+class TestUserCohortAPI(TestCase):
```

Comment:
https://www.django-rest-framework.org/api-guide/testing/#api-test-cases instead of TestCase

Reply:
thanks!